packages/okta-angular/README.md

```diff
@@ -1,62 +1,100 @@
 # Okta Angular SDK
-The Okta Angular SDK is a wrapper around [Okta Auth JS](https://github.com/okta/okta-auth-js), that builds on top of Okta's [OpenID Connect API](https://developer.okta.com/docs/api/resources/oidc.html).
+An Angular (4+) wrapper around [Okta Auth JS](https://github.com/okta/okta-auth-js), that builds on top of Okta's [OpenID Connect API](https://developer.okta.com/docs/api/resources/oidc.html).
 
 This library currently supports:
   - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
 
 ## Getting Started
 * If you do not already have a **Developer Edition Account**, you can create one at [https://developer.okta.com/signup/](https://developer.okta.com/signup/).
-* If you don't have an Angular app, or are new to Angular, please start with the [Angular Quickstart](https://angular.io/guide/quickstart) guide. It will walk you through the creation of an Angular app, creating routes, and other application development essentials.
 
-### Add an OpenID Connect Client in Okta
-In Okta, applications are OpenID Connect clients that can use Okta Authorization servers to authenticate users.  Your Okta Org already has a default authorization server, so you just need to create an OIDC client that will use it.
-* Log into the Okta Developer Dashboard, click **Applications** > **Add Application**.
-* Choose **Single Page App (SPA)** as the platform, then submit the form the default values, which should look like this:
-
-| Setting             | Value                                          |
-| ------------------- | ---------------------------------------------- |
-| App Name            | My SPA App                                     |
-| Base URIs           | http://localhost:{port}                        |
-| Login redirect URIs | http://localhost:{port}/implicit/callback      |
-| Grant Types Allowed | Implicit                                       |
-
-After you have created the application there are two more values you will need to gather:
-
-| Setting       | Where to Find                                                                  |
-| ------------- | ------------------------------------------------------------------------------ |
-| Client ID     | In the applications list, or on the "General" tab of a specific application.   |
-| Org URL       | On the home screen of the developer dashboard, in the upper right.             |
-
-
-These values will be used in your Angular application to setup the OpenID Connect flow with Okta.
+| Helpful Links | Description |
```

Comment:
This tables feels odd below an orphaned single bullet :D 

I think we should bring in this prereq information to help get folks closer to ready: https://github.com/okta/samples-js-angular/tree/master/okta-hosted-login#prerequisites

And then play with the formatting to make this table feel less like a hanger-on.  Maybe it should just be a list of links?